### PR TITLE
Update Org Browser article

### PR DESCRIPTION
**Pull request by hbc-bindu**
### What does this PR do?
Add the ability to retrieve all of the metadata of the same type and update the screenshot.

### What issues does this PR fix or reference?
@W-7222242@

### CODE REVIEW

File: docs/_articles/en/user-guide/org-browser.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ The Org Browser displays the available metadata types and their corresponding co
 5. Metadata component
 6. Refresh components for metadata type
 7. Retrieve source for metadata component
+8. Retrieve source for metadata type
```
Comment (brpowell) [inline docs/_articles/en/user-guide/org-browser.md]:
```suggestion
8. Retrieve source for all components of metadata type
```

**Comment by brpowell** [comment]
Not directly related to these changes but since we're updating the article, on [this line](https://github.com/forcedotcom/salesforcedx-vscode/pull/1971/files#diff-7916c2d2c884dbfad4b6885401fd2abcL6) where we say

> This feature makes it easier and simpler to retrieve metadata source...

Do we have to say both _easier_ and _simpler_? They are synonyms of each other.

File: docs/_articles/en/user-guide/org-browser.md [hunk 2]
```diff
@@ -34,9 +35,14 @@ You can refresh metadata at org level, for a type, for folders in a type, and fo
 
 ![Metadata type with folders](./images/org_browser_folders.png)
 
-## Retrieving Components
+## Retrieving Metadata
 
-You can retrieve a component to your local project by clicking the retrieve button **(7)** next to the component name. Currently, the component is retrieved into your default package directory, which is defined in the [sfdx-project.json](./en/getting-started/first-project#the-sfdx-projectjson-file) file.
+You can retrieve a metadata component or all metadata of the same type. Click retrieve icon next to:
```
Comment (brpowell) [inline docs/_articles/en/user-guide/org-browser.md]:
```suggestion
You can retrieve a single metadata component, or all components of the same type. Click retrieve icon next to:
```

File: docs/_articles/en/user-guide/org-browser.md (hunk 1 above)
Comment (hbc-bindu) [reply docs/_articles/en/user-guide/org-browser.md]: Done

File: docs/_articles/en/user-guide/org-browser.md (hunk 2 above)
Comment (hbc-bindu) [reply docs/_articles/en/user-guide/org-browser.md]: Done

Comment (hbc-bindu) [reply docs/_articles/en/user-guide/org-browser.md]: @smaddox-sf , @brpowell , let me know if you have any more comments...

**Pull request merged**